keyboards/essemi/fs026/mcuconf.h

```diff
@@ -0,0 +1,162 @@
+/*
+    Copyright (C) 2021 essemi
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#ifndef MCUCONF_H
+#define MCUCONF_H
+
+/**
+ * @name    Internal clock sources
+ * @{
+ */
+#define ES32_PLLCLK 72000000
+#define ES32_HRCCLK 48000000
+#define ES32_LSECLK 32768
+
+/*
+ * HAL driver system settings.
+ */
+#define ES32_NO_INIT FALSE
+#define ES32_MHSI_ENABLED TRUE
+#define ES32_FHSI_ENABLED FALSE
+#define ES32_LSI_ENABLED FALSE
+#define ES32_HSE_ENABLED TRUE
+#define ES32_LSE_ENABLED FALSE
+#define ES32_PLL_ENABLED TRUE
+#define ES32_MAINCLKSRC ES32_MAINCLKSRC_PLL
+#define ES32_HSE_STATE ANCTL_HSECR0_HSEON
+#define ES32_PLLSRC ES32_PLLSRC_HSE
+#define ES32_PLLDIV_VALUE 2
+#define ES32_PLLMUL_VALUE 12 // The allowed range is 12,16,20,24.
+#define ES32_HPRE 1
+#define ES32_PPRE1 1
+#define ES32_PPRE2 1
+#define ES32_USBPRE ES32_USBPRE_DIV1P5
+
+/*
+ * EXTI driver system settings.
+ */
+#define ES32_IRQ_EXTI0_PRIORITY 6
+#define ES32_IRQ_EXTI1_PRIORITY 6
+#define ES32_IRQ_EXTI2_PRIORITY 6
+#define ES32_IRQ_EXTI3_PRIORITY 6
+#define ES32_IRQ_EXTI4_PRIORITY 6
+#define ES32_IRQ_EXTI5_9_PRIORITY 6
+#define ES32_IRQ_EXTI10_15_PRIORITY 6
+#define ES32_IRQ_EXTI16_PRIORITY 6
+#define ES32_IRQ_EXTI17_PRIORITY 6
+#define ES32_IRQ_EXTI18_PRIORITY 6
+#define ES32_IRQ_EXTI19_PRIORITY 6
+
+/*
+ * GPT driver system settings.
+ */
+#define ES32_TIM_MAX_CHANNELS 4
+#define ES32_GPT_USE_TIM1 FALSE
+#define ES32_GPT_USE_TIM2 FALSE
+#define ES32_GPT_USE_TIM3 FALSE
+#define ES32_GPT_USE_TIM4 FALSE
+#define ES32_GPT_TIM1_IRQ_PRIORITY 7
+#define ES32_GPT_TIM2_IRQ_PRIORITY 7
+#define ES32_GPT_TIM3_IRQ_PRIORITY 7
+#define ES32_GPT_TIM4_IRQ_PRIORITY 7
+
+/*
+ * ICU driver system settings.
+ */
+#define ES32_ICU_USE_TIM1 FALSE
+#define ES32_ICU_USE_TIM2 FALSE
+#define ES32_ICU_USE_TIM3 FALSE
+#define ES32_ICU_USE_TIM4 FALSE
+#define ES32_ICU_TIM1_IRQ_PRIORITY 7
+#define ES32_ICU_TIM2_IRQ_PRIORITY 7
+#define ES32_ICU_TIM3_IRQ_PRIORITY 7
+#define ES32_ICU_TIM4_IRQ_PRIORITY 7
+
+/*
+ * PWM driver system settings.
+ */
+#define ES32_PWM_USE_ADVANCED FALSE
+#define ES32_PWM_USE_TIM1 FALSE
+#define ES32_PWM_USE_TIM2 FALSE
+#define ES32_PWM_USE_TIM3 FALSE
+#define ES32_PWM_USE_TIM4 FALSE
```

Comment:
These are written as TIM1...TIM4, but the backlight is called GP16C4T2.
The defines don't match https://github.com/ChibiOS/ChibiOS-Contrib/blob/chibios-21.11.x/os/hal/ports/ES32/LLD/TIMv1/hal_pwm_lld.h

Please fix up this file so that ALL item names are actually correct, not just the PWM ones.